examples/compositor/xdg_shell.c

```diff
@@ -0,0 +1,210 @@
+#include <assert.h>
+#include <wayland-server.h>
+#include <wlr/util/log.h>
+#include <stdlib.h>
+#include "compositor.h"
+#include "compositor/protocols/xdg-shell.h"
+
+static void resource_destructor(struct wl_client *client,
+		struct wl_resource *resource) {
+   wl_resource_destroy(resource);
+}
+
+static void xdg_toplevel_set_parent(struct wl_client *client,
+		struct wl_resource *resource, struct wl_resource *parent_resource) {
+	wlr_log(L_DEBUG, "TODO: toplevel set parent");
+}
+
+static void xdg_toplevel_set_title(struct wl_client *client,
+		struct wl_resource *resource, const char *title) {
+	wlr_log(L_DEBUG, "TODO: toplevel set title");
+}
+
+static void xdg_toplevel_set_app_id(struct wl_client *client,
+		struct wl_resource *resource, const char *app_id) {
+	wlr_log(L_DEBUG, "TODO: toplevel set app id");
+}
+
+static void xdg_toplevel_show_window_menu(struct wl_client *client,
+		struct wl_resource *resource, struct wl_resource *seat, uint32_t serial,
+		int32_t x, int32_t y) {
+	wlr_log(L_DEBUG, "TODO: toplevel show window menu");
+}
+
+static void xdg_toplevel_move(struct wl_client *client, struct wl_resource *resource, struct wl_resource *seat_resource, uint32_t serial) {
+	wlr_log(L_DEBUG, "TODO: toplevel move");
+}
+
+static void xdg_toplevel_resize(struct wl_client *client, struct wl_resource
```

Comment:
Don't split parameters across two lines, move the whole declaration to the next line. Take `client` with if it looks bad.